phoenicis-repository/src/main/java/org/phoenicis/repository/RepositoryException.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2015-2017 PÂRIS Quentin
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+package org.phoenicis.repository;
+
+public class RepositoryException extends RuntimeException {
+    public RepositoryException(String message) {
+        super(message);
+    }
+
+    public RepositoryException(String message, Exception e) {
```

Comment:
Take a look at the extended class https://docs.oracle.com/javase/7/docs/api/java/lang/RuntimeException.html. The constructor you call with `super(message, e);` takes a `Throwable` as its second argument.